crates/walrus-service/bin/backup.rs

```diff
@@ -21,12 +21,14 @@ use walrus_service::{
 
 /// Manage and run Walrus backup nodes
 #[derive(Parser)]
-#[clap(rename_all = "kebab-case")]
-#[clap(name = env!("CARGO_BIN_NAME"))]
-#[clap(version = VERSION)]
+#[command(
+    name = env!("CARGO_BIN_NAME"),
+    version = VERSION,
+    rename_all = "kebab-case",
+)]
 #[derive(Debug)]
 struct Args {
-    #[clap(long, short, help = "Specify the config file path to use")]
+    #[arg(long, short, help = "Specify the config file path to use")]
     config: PathBuf,
     #[clap(
```

Comment:
There is another instance of `clap` here:
```suggestion
    #[arg(
```